types/V1.mli

```diff
@@ -51,8 +51,8 @@ module type TIME = sig
   type +'a io
   (** The type for potentially blocking I/O operation *)
 
-  val sleep: float -> unit io
-  (** [sleep nsec] Block the current thread for. {b FIXME:} remove float. *)
+  val sleep: int64 -> unit io
+  (** [sleep n] Block the current thread for n nanoseconds. *)
```

Comment:
signed (no feelings about it, but that's what I can rely on using Caml's Int64).

Behaviour for negative values is implementation-defined - I couldn't find the source for lwt, but [our xen implementation](https://github.com/mirage/mirage-platform/blob/ed19a1a318e944a7020239fa01c0b674b02cf471/xen/lib/time.ml#L72) skips a negative sleep.